akka-persistence-typed/src/test/java/akka/persistence/typed/javadsl/PersistentActorFailureTest.java

```diff
@@ -19,9 +18,9 @@
 
 import java.time.Duration;
 
-import static akka.persistence.typed.scaladsl.PersistentBehaviorFailureSpec.conf;
+import static akka.persistence.typed.scaladsl.EventSourcedBehaviorFailureSpec.conf;
 
-class FailingPersistentActor extends PersistentBehavior<String, String, String> {
+class FailingPersistentActor extends EventSourcedBehavior<String, String, String> {
```

Comment:
I guess it would be more correct to say event sourced actor now

Reply:
Well, the modules are Akka-persistence*, are we wanting to expose it everywhere and get rid of so much of the *Persistence* notion? That is a q I had during this.

Reply:
I think the module name is the least problem because you'd put that once in your build config and then forget about it, the important thing is that we are consistent in the docs and the APIs and don't use different names for the same thing.

Reply:
I don't think we have to change module and package name. Event sourcing is one type of persistence. We might add other types, and at least put the APIs for those in same module.

Reply:
Oh sorry, I wasn't proposing a module/package rename, just asking about the creep of `EventSourced*` over `Persistence*`.

Thanks.